demos/classification_demo/cpp/README.md

```diff
@@ -20,7 +20,7 @@ The list of models supported by the demo is in `<omz_dir>/demos/classification_d
 This file can be used as a parameter for [Model Downloader](../../../tools/downloader/README.md) and Converter to download and, if necessary, convert models to OpenVINO Inference Engine format (\*.xml + \*.bin).
 
 An example of using the Model Downloader:
-
+-
```

Comment:
```suggestion

```

Reply:
This should be empty line: https://github.com/openvinotoolkit/open_model_zoo/commit/c7d1a8b242a99793e1ade36fa219d3f4d780f6d2. Strangely github interpreted the suggestion as line removal.

Reply:
Restored empty line.